tests/functional/relation_tests/test_relation_type_change.py

```diff
@@ -34,31 +40,86 @@ def error_message(self) -> str:
 
 relations = [
     Model(models.VIEW, "view"),
-    Model(models.TABLE, "table"),
-    Model(models.DYNAMIC_TABLE, "dynamic_table"),
+    Model(models.TABLE, "table", "default"),
+    # to be activated upon merge of dynamic table support PR
+    # Model(models.DYNAMIC_TABLE, "dynamic_table", "default"),
+    # Model(models.DYNAMIC_ICEBERG_TABLE, "dynamic_table", "iceberg"),
+    Model(models.ICEBERG_TABLE, "table", "iceberg"),
+    Model(models.ICEBERG_INCREMENTAL_TABLE, "table", "iceberg", incremental=True),
```

Comment:
Technically the way these tests work is by grabbing what Snowflake calls it, which is going to be a table, so it makes sense to use that value here, despite the fact it's from an incremental model